oscap_report/cli.py

```diff
@@ -16,7 +16,8 @@
                " the --output option is specified it will be written to standard output.")
 LOG_LEVES_DESCRIPTION = (
     "LOG LEVELS:\n"
-    "\tDEBUG - Detailed information, typically of interest only when diagnosing problems.\n"
+    "\tDEBUG - Detailed information, typically of interest only when diagnosing problems. "
+    "(does not minimize html content)\n"
```

Comment:
I don't really like mixing up multiple unrelated behavior controls under a single switch. While it might seem that these two debug options are related, they in fact are not. The log option should never control the content of the resulting file. It is better to introduce another option, connected to the report (i.e. real product of the tool) itself.

Reply:
I agree.

Reply:
Ok, I will introduce a new format of report for debugging purposes. For example `-f DEBUG-HTML`